src/schema/nmdc.yaml

```diff
@@ -404,6 +411,11 @@ classes:
       - lbceq
 
     slot_usage:
+      id: 
+        structured_pattern:
+          syntax: "{id_nmdc_prefix}{id_type_sample}{id_shoulder}{id_generated}"
```

Comment:
is there any weirdness about the  id_shoulder being greedy or not greedy?

Reply:
That's a good catch; I think the `[a-z]*` in the should part should be lazy. I'll fix.

`interpolated: true` means that values from `settings` should be interpolated into the `syntax` in order to construct the final regular expression: https://linkml.io/linkml/schemas/constraints.html#structured-patterns

Reply:
thanks. On inspection, I think your fix won't be necessary because a numeric character `[0-9]` is required at the shoulder end and is in no danger of being slurped up by `[a-z]*`. My only other comment is that perhaps the prefix should be "nmdc" and the ":" put as a literal into the syntax, but none of this is blocking. I approve.